sssd_test_framework/roles/ad.py

```diff
@@ -833,6 +847,65 @@ def modify(
         }
 
         self._modify(attrs)
+
+        # Password changes require a different command group so special
+        # handling is needed
+        if password is not None:
+            self.reset(str(password))
+
+        return self
+
+    def reset(
+        self,
+        password: str = "Secret123",
+    ) -> ADUser:
+        """
+        Reset user password
+
+        Parameters that are not set are ignored.
```

Comment:
Redundant, the only parameter is password with a default value.

Reply:
removing